### ISSUE TITLE
Is the operand table valid?

### ISSUE THREAD

**Issue by CeciAc** [MicrosoftDocs/sql-docs#2293]
Hi team, 
I moved this from here: https://github.com/MicrosoftDocs/sql-docs.de-de/issues/85
the contributor suggests an adition to the original documentation
best

@Tornhoof commented 4 hours ago
In the table above, bigint is not listed in the right operands of bigint, the description does not exclude it though and something like this:

```
declare @rside as bigint
declare @lside as bigint
set @lside = 922337203685477580
set @rside = 9223372036854775807
SELECT ( @lside & @rside)
GO
```

works in SQL Server 2017.
You might want to rework the table, either you properly list all valid combinations, then you have a NxM matrix or you just describe the exceptions.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b730b2bb-57ee-aba8-247f-0fa63114c88b
* Version Independent ID: 36fd9a35-e182-a272-a9f4-f2da1eec7d17
* Content: [Bitwise Operators (Transact-SQL) - SQL Server](https://docs.microsoft.com/en-us/sql/t-sql/language-elements/bitwise-operators-transact-sql?view=sql-server-2017#feedback)
* Content Source: [docs/t-sql/language-elements/bitwise-operators-transact-sql.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/language-elements/bitwise-operators-transact-sql.md)
* Product: **sql**
* Technology: **t-sql**
* GitHub Login: @rothja
* Microsoft Alias: **jroth**

**Comment by WilliamAntonRohm** [comment]
@CeciAc -- closing as this issue has addressed in documentation PR #2299.

I notifed the originator (Tornhoof) in the source German issue.

**Comment by redwiking** [comment]
Hi,
Isn't the table at the end of this article still a little bit misleading? 
The only constraint for bitwise operations is not to use two binary string data type operands together, all other combinations are allowed. When there are different types of operands, one with the lower precedence is converted to a higher precedence type, and then a bitwise operation is performed. 
So, in my opinion, the table just introduces confusion.
